linera-service/src/cli_wrappers/wallet.rs

```diff
@@ -374,6 +377,9 @@ impl ClientWrapper {
         let port = port.into().unwrap_or(8080);
         let mut command = self.command().await?;
         command.arg("service");
+        if self.deterministic {
+            command.arg("--listener-skip-process-inbox");
+        }
```

Comment:
Why not pass this in only to `run_node_service`, rather than having it as a field?

Also, I'd call it `skip_process_inbox` rather than `deterministic`, since that's more specific, and there are potentially other sources of non-determinism.

Reply:
Ok, makes sense.